packages/playwright-core/src/server/chromium/crNetworkManager.ts

```diff
@@ -735,7 +770,8 @@ class ResponseExtraInfoTracker {
     if (response && responseExtraInfo) {
       response.setResponseHeadersSize(responseExtraInfo.headersText?.length || 0);
       response.setRawResponseHeaders(headersObjectToArray(responseExtraInfo.headers, '\n'));
-      info.responseReceivedExtraInfo[index] = undefined;
+      response.setRawStatus(responseExtraInfo.statusCode, getStatusText(responseExtraInfo.headersText));
```

Comment:
in case of HTTP/2 or QUIC headersText is empty

Reply:
Yes. What do you suggest?